docs/source/API/core/view/view.rst

```diff
@@ -565,6 +625,35 @@ Assignment Examples
     View<int*, HostSpace> a12 = View<int*, CudaSpace>("A12",N); // Error: non-assignable memory spaces
     View<int*, HostSpace> a13 = View<int*, CudaHostPinnedSpace>("A13",N); // OK
 
+.. _api-view-natural-mdspans:
+
+Natural mdspans
+---------------
+
+.. versionadded:: 4.4.0
```

Comment:
Since 4.4 is not deployed, would it mean that this documentation will on hold till 4.4 is released?

Reply:
No I think we should merge now since the changes are in develop. We could label with something like (in development) or something but that would mean we would have to remember to change it back

Reply:
I think it's fine.